src/components/TileGrid/TileGrid.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import SquareTile from './Tile/SquareTile';
+import StackGrid, { transitions } from "react-stack-grid";
+import RectTile from './Tile/RectTile';
+
+const { scaleDown } = transitions;
+
+const TileGrid = () => {
+    return (
+        <StackGrid
+            columnWidth={230}
+            gutterWidth={20}
+            gutterHeight={20}
+            appear={scaleDown.appear}
+            appeared={scaleDown.appeared}
+            enter={scaleDown.enter}
+            entered={scaleDown.entered}
+            leaved={scaleDown.leaved}
+        >
+            <SquareTile title="ACM Kickoff" date="Today" time="8:30PM" num={Math.floor(Math.random()*6)}/>
```

Comment:
Once we get real data (or maybe just a dummy json file) we should change this to be something nicer like 
```
events.map((item, index) => {
     // probably find a way to randomize between RectTile and SquareTile too
     <RectTile event={item}/>
}
```